angelssmelting/prototypes/buildings/induction-furnace.lua

```diff
@@ -232,9 +232,9 @@ data:extend(
           pipe_covers = pipecoverspictures(),
           base_level = 1,
           pipe_connections = {{ position = {2, -3} }}
-        }
+        },
+        off_when_no_fluid_recipe = true,
       },
```

Comment:
Same fix should be applied to `induction-furnace-2` https://github.com/Arch666Angel/mods/blob/bb220ff974e4aa8e1a24893cb059d0e06a8c40bf/angelssmelting/prototypes/buildings/induction-furnace.lua#L471-L473